tests/unit/latency-monitor.tcl

```diff
@@ -143,7 +143,11 @@ tags {"needs:debug"} {
 } ;# tag
 
     test {LATENCY of expire events are correctly collected} {
-        r config set latency-monitor-threshold 20
+<<<<<<< Updated upstream
+        r config set latency-monitor-threshold 5
+=======
+        r config set latency-monitor-threshold 2
+>>>>>>> Stashed changes
```

Comment:
Why 2/5 and not '1'? I would be happy if we just had a way to report any action to latency monitor (zero will currently disable it).

Reply:
Oops, I added a merge conflict... No reason. Let's make it 1.

```suggestion
        r config set latency-monitor-threshold 1
```